lib/requirejs/require.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Non-minified: Perhaps it will be easier to debug some things. But you're right that we change it at any moment.

npm: I understand it as a dependency (not as a dev dependency). Just like CKEditor-core or plugins, it's needed to run a dev version of the project. But I would choose the npm also because of the maintenance - we won't need to manually upgrade the lib or write a grunt task for that. These are small things, but remembering about stuff like this (e.g. upgrading jQ) in CKEditor 4 is tiring.

BTW. do we anticipate having more libraries like this one?